### DISCUSSION TITLE
Rust Binary Installation Action for Installing `mdbook` in CD?

### DISCUSSION THREAD

**Discussion by kevinmatthes** [sourcefrog/cargo-mutants#202] [Ideas]
I would like to suggest the following GitHub Action in order to optimise the book deployment workflow:  https://github.com/taiki-e/install-action.

This Action will download the desired binary, install it into the range of `$PATH` and store a copy of it in a cache for faster access in case it would be required again in the near future.  The change to the workflow definition would be very straightforward, the approach works well in all repositories I maintain and / or contribute to, and the Action is highly actively maintained.

What do you think, @sourcefrog?

**Comment by sourcefrog** [comment]
I don't perceive book deployment speed as a particular pain point, but it seems like a good change, and a good initial step towards the larger goal of installing cargo-mutants this way.

**Reply by kevinmatthes** [reply]
Okay, I am about to prepare the corresponding PR. :+1:

**Reply by kevinmatthes** [reply]
I just submitted #203.

**Reply by kevinmatthes** [reply]
As #203 is merged, now, I will mark this discussion as resolved.
